js/ui/components/clubhouse.js

```diff
@@ -225,6 +225,24 @@ var ClubhouseAnimator = class ClubhouseAnimator {
     }
 };
 
+var QuestBannerPosition = class QuestBannerPosition {
+    constructor() {
+        this._atBottom = false;
```

Comment:
nitpick: I'd rename this to position, and let the value to be "like an enum" like in here https://github.com/endlessm/gnome-shell/blob/master/js/misc/soundServer.js#L36-L40

Reply:
Great idea, thanks!

Reply:
After a second thought, I really prefer a boolean here. Because there are only two cases, at bottom or not (ie at top). An enum-like variable will make the check complicated, with a switch/case statement.